pager/src/main/java/com/google/accompanist/pager/Pager.kt

```diff
@@ -97,51 +108,62 @@ private val Measurable.page: Int
 fun HorizontalPager(
     state: PagerState,
     modifier: Modifier = Modifier,
+    reverseLayout: Boolean = false,
     @IntRange(from = 1) offscreenLimit: Int = 1,
-    content: @Composable PagerScope.(page: Int) -> Unit
+    decayAnimationSpec: DecayAnimationSpec<Float> = defaultDecayAnimationSpec(),
+    snapAnimationSpec: AnimationSpec<Float> = spring(stiffness = SnapSpringStiffness),
```

Comment:
Here's is an idea: since your internal PagerState.fling uses only public API available (if not, it's still a good idea :) ) what you can do here is to replace two animation specs with one FlingBehaviour, e.g.
```
object PagerDefaults {
     @Composable
     fun defaultPagerFlingConfig(state: PagerState, decayAnimationSpec: DecayAnimationSpec<Float>, snapAnimationSpec: AnimationSpec<Float>): FlingConfig { ... }
}
```

Or similar, then you can just open the whole flingbehaviour for overriding (make FlingBehaviour a param in HorizontalPager), so people can:
1. Use default behavior by providing nothing
2. Use default with tweaked params
3. Use their own sophisticated or slightly different fling behaviour by providing their own instance.
4. The API for customization will be consistent with LazyColumn and friends.

What do you think?

Reply:
Yeah I like it, I'll follow up after #254